bazel/init/stage_1.bzl

```diff
@@ -176,6 +176,21 @@ def stage_1():
         ],
     )
 
+    maybe(
```

Comment:
can you also put a comment just before grpc_deps() is invoked stating something like:
```
# IMPORTANT: grpc_deps() pulls in boringssl as a WORKSPACE dependency. In order to apply patches
# to boringssl, we define boringssl BEFORE grpc_deps is invoked - that way, our version will be picked
# over the default one.
# You must manually update boringssl when grpc is updated. If ARM support is added upstream, we may
# be able to remove the patches and the work here.
```

Reply:
Done - https://github.com/enfabrica/enkit/pull/1104/files#diff-59c9ccc93dd10c39d3c243e5bf9d2a6cf5b596171078a3a12b052cc257b106b1R87